### PR TITLE
Gps level

### PR DESCRIPTION
**Pull request by trevormccrt**

### CODE REVIEW

**Comment by sergei1152** [comment]
Why does this PR have the commit from the previous PR? They should be seperate. Please get rid of it ( the commit with gains import and export) otherwise it'll cause a merge conflict. Make sure you switch to master before creating a new branch so that this won't happen.

File: app/views/StatusView.js [hunk 1]
```diff
@@ -141,6 +163,13 @@ module.exports=function(Marionette,$){
         return true;
       }
       return false;
+    },
+    validGPS:function(gps)
```
Comment (sergei1152) [inline app/views/StatusView.js]: Put this function in the validator module

File: warggroundstation [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 684ad1dfd95a8025f05ba8f5afdead6c17c57f4d
```
Comment (sergei1152) [inline warggroundstation]: Get rid of this thing. Netbeans sucks.

Comment (CoatedMoose) [inline warggroundstation]: As mentioned on other PR, this has nothing to do with netbeans and is a
valid git file that should be committed (when using git submodules).

**Pull request closed without merging**